src/proteus/dswx_hls.py

```diff
@@ -2939,6 +2938,13 @@ def _get_tile_srs_bbox(tile_min_y_utm, tile_max_y_utm,
               Tile maximum X-coordinate (polygon SRS)
     """
 
+    # forces returned values from TransformPoint() to be (x, y, z)
+    # rather than (y, x, z) for geographic SRS
+    if polygon_srs.IsGeographic():
+        try:
+            polygon_srs.SetAxisMappingStrategy(osr.OAMS_TRADITIONAL_GIS_ORDER)
+        except AttributeError:
+            pass
```

Comment:
If you happen to hit this `except`, I would think you'd want to at least log that it occurred.

Reply:
Thank you, @collinss-jpl ! Just updated the code to log it as a warning message.